pkg/sql/sem/builtins/replication_builtins.go

```diff
@@ -385,4 +389,84 @@ var replicationBuiltins = map[string]builtinDefinition{
 			Volatility: volatility.Volatile,
 		},
 	),
+	"crdb_internal.split_at": makeBuiltin(
+		tree.FunctionProperties{
+			Category:         builtinconstants.CategorySystemRepair,
+			Undocumented:     true,
+			DistsqlBlocklist: true,
+		},
+		tree.Overload{
+			Types: tree.ParamTypes{
+				{Name: "key", Typ: types.Bytes},
+				{Name: "ttl", Typ: types.Interval},
+			},
+			ReturnType: tree.FixedReturnType(types.Void),
+			Fn: func(ctx context.Context, evalCtx *eval.Context, args tree.Datums) (tree.Datum, error) {
+				if err := evalCtx.SessionAccessor.CheckPrivilege(
+					ctx, syntheticprivilege.GlobalPrivilegeObject, privilege.REPAIRCLUSTER,
+				); err != nil {
+					return nil, err
+				}
+				key := roachpb.Key(tree.MustBeDBytes(args[0]))
+				ttl := tree.MustBeDInterval(args[1])
+				expiration := evalCtx.Txn.DB().Clock().Now().Add(ttl.Nanos(), 0)
+				return tree.DVoidDatum, evalCtx.Txn.DB().AdminSplit(ctx, key, expiration)
+			},
+			Info:       "Splits at an *arbitrary* byte key.",
+			Volatility: volatility.Volatile,
+		},
+	),
+	"crdb_internal.scatter": makeBuiltin(
+		tree.FunctionProperties{
+			Category:         builtinconstants.CategorySystemRepair,
+			Undocumented:     true,
+			DistsqlBlocklist: true,
+		},
+		tree.Overload{
+			Types: tree.ParamTypes{
+				{Name: "key", Typ: types.Bytes},
+			},
+			ReturnType: tree.FixedReturnType(types.Void),
+			Fn: func(ctx context.Context, evalCtx *eval.Context, args tree.Datums) (tree.Datum, error) {
+				if err := evalCtx.SessionAccessor.CheckPrivilege(
+					ctx, syntheticprivilege.GlobalPrivilegeObject, privilege.REPAIRCLUSTER,
+				); err != nil {
+					return nil, err
+				}
+				key := roachpb.Key(tree.MustBeDBytes(args[0]))
+				_, err := evalCtx.Txn.DB().AdminScatter(ctx, key, 0)
+				return tree.DVoidDatum, err
+			},
+			Info:       "Scatters the passed arbitrary key",
+			Volatility: volatility.Volatile,
+		},
+		tree.Overload{
+			Types: tree.ParamTypes{
+				{Name: "key", Typ: types.Bytes},
+				{Name: "end_key", Typ: types.Bytes},
+			},
+			ReturnType: tree.FixedReturnType(types.Void),
```

Comment:
Would be useful to return some portion of the RangeInfo contained in the AdminScatterResponse?

Reply:
🤷 Can add it when we need it?